src/client/js/Utils/Exporters.js

```diff
@@ -60,12 +65,21 @@ define(['js/Loader/ProgressNotification'], function (ProgressNotification) {
                         progress: 100
                     });
                 } else {
+
                     progress.note.update({
                         message: '<strong>Exported </strong> models <a href="' +
                         result.downloadUrl + '" target="_blank">' + result.fileName + '</a>',
                         progress: 100,
-                        type: 'success'
+                        type: 'success',
+                        icon: 'glyphicon glyphicon-copy'
                     });
+
+                    //TODO it is not the nicest way, we might want to add it as an option to the notification itself
+                    $(progress.note.$ele).find('.glyphicon-copy').attr('title', 'copy blobhash of export to clipboard');
+                    $(progress.note.$ele).find('.glyphicon-copy').attr('data-clipboard-text', result.hash);
+                    $(progress.note.$ele).find('.glyphicon-copy').css('cursor', 'copy');
+                    $(progress.note.$ele).find('.glyphicon-copy').addClass('btn btn-xs');
+                    new Clipboard($(progress.note.$ele).find('.glyphicon-copy')[0]);
```

Comment:
Minor note, instead of finding the icon five times you could either make a local variable or chain the setters.